helper/schema/resource_importer.go

```diff
@@ -83,3 +84,30 @@ func ImportStatePassthrough(d *ResourceData, m interface{}) ([]*ResourceData, er
 func ImportStatePassthroughContext(ctx context.Context, d *ResourceData, m interface{}) ([]*ResourceData, error) {
 	return []*ResourceData{d}, nil
 }
+
+func ImportStatePassthroughWithIdentity(idAttributePath string) StateContextFunc {
```

Comment:
Since this is an exported function, we should add some general docs here (use when you need to support identity AND id-only refresh).

Since identity attributes are "flat", the only real accepted path for `idAttributePath` here should be an attribute name, but there's no real way to enforce it, just document 😄. It also must be a string identity attribute, which might be worth noting.

Reply:
Oh, of course – totally missed those docs 😅

Reply:
Oh, of course – totally missed those docs 😅